appveyor.yml

```
@@ -19,9 +19,13 @@ environment:
     - STACK_YAML: stack-lts-6.yaml
 
 install:
-  - choco install -y haskell-stack --version %STACK_VERSION%
-  - stack setup > nul
+  # http://help.appveyor.com/discussions/problems/6312-curl-command-not-found
+  - set PATH=C:\Program Files\Git\mingw64\bin;%PATH%
+
+  - curl -sS -ostack.zip -L --insecure "https://github.com/commercialhaskell/stack/releases/download/v%STACK_VERSION%/stack-%STACK_VERSION%-windows-x86_64.zip"
```

I try to use long option names for commands that are run non-interactively to better document them:

```suggestion
  - curl --silent --show-error --output stack.zip --location --insecure "https://github.com/commercialhaskell/stack/releases/download/v%STACK_VERSION%/stack-%STACK_VERSION%-windows-x86_64.zip"
```

Also, is the `--insecure` necessary when fetching from `github.com`?  I thought that was only for domains with an invalid certificate